### ISSUE TITLE
auto_more hides every messages

### ISSUE THREAD

**Issue by Adrien2002** [angband/angband#4736]
I just built the game from source after cloning the git repo and I realized that when enabling the auto_more option, I don't have any message, like when I'm hitting a wall or walking on stuff or anything that would help me understanding what is going on in the game. Pressing CTRL + P shows me every messages as usual, they just don't appear while playing.
Is that a new feature ? Or is that a bug ?

**Comment by NickMcConnell** [comment]
It's a (slightly controversial) new feature, based on the assumption that people will only use auto_more if they have a message window open.  If enough people dislike it it will probably get reverted.

**Comment by Adrien2002** [comment]
I can't say I like it or not, I've always read the messages on top of the screen, keeping an eye on the full history from time to time when I have a doubt but having a second window would reduce my view of the dungeon which is always in fullscreen. Auto_more never stops you from playing but you have to be more careful so when approaching danger, I slow down and take my time in every turn, using history to check what happens at every turn pressing CTRL + P. I got used to it over the years.

Not having any message is making me feel like I'm blind. I realize how important those messages are to me, even they appear quickly and disappear, I rapidly identify them and know what's happening but not having them is totally losing me, like... I just see a screen full of ASCII chars wondering what's happening in the game.

So, for me, if I can give my opinion, it would be :

Adding the option to disable the message on the main screen to only keep a second window full of history -> **YES**
Replacing the original auto_more option with this feature, forcing people to use a second window -> **NO**

Now, that's just one opinion of one Angband player, not everyone will share my thoughts.

**Comment by NickMcConnell** [comment]
It's difficult to assess the popularity of this - I have had two complaints (including this one) but the initial push was from consensus on angband.live.  I'm going to call this an Ongoing issue, and we'll wait for more data.

**Comment by shaggyfrog** [comment]
I just updated from 4.2.0 to 4.2.2 and I also dislike how this feature has been changed... definitely against the "principle of least surprise". I wasn't even sure what the problem was at first. I go back and forth between having a secondary message window or just a single window, and in both cases I want to see the last message in the main window, and I don't want to turn off auto_more, but I guess I have to now.
